crm_task/views/project_task.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    
+    <!-- FORM -->
+    <record id="project_task_form2" model="ir.ui.view">
+        <field name="name">project.task.form.2</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="hr_timesheet.view_task_form2_inherited"/>
```

Comment:
As you're working on 'user_id' you don't need to use ref="hr_timesheet.view_task_form2_inherited" because this field is in "project.view_task_form2". Right?